syntax = "proto3";

package chat;

message ChatRequest {
    string msg = 1;
    string model = 2;
}

message ChatResponse {
    string msg = 1;
}

service ChatService {
    rpc GetChat(ChatRequest) returns (ChatResponse);
}
